android/app/src/main/java/com/cleverraven/cataclysmdda/SplashScreen.java

```diff
@@ -28,7 +28,7 @@
     private static final int INSTALL_DIALOG_ID = 0;
     private ProgressDialog installDialog;
 
-    public CharSequence[] mSettingsNames = { "Software rendering", "Force fullscreen", "Trap Back button" };
+    public CharSequence[] mSettingsNames;
```

Comment:
These untranslated strings are used as settings key names when calling `android_get_default_setting` from C++ code:

```
    add( "SOFTWARE_RENDERING", "graphics", to_translation( "Software rendering" ),
         to_translation( "Use software renderer instead of graphics card acceleration.  Requires restart." ),
         // take default setting from pre-game settings screen - important as both software + hardware rendering have issues with specific devices
         android_get_default_setting( "Software rendering", false ),
         COPT_CURSES_HIDE
       );
```

With changes from current PR when device language would be set to non-English - settings won't be found.

You can probably add something like `public CharSequence[] mSettingsIds;` and use these Ids in `android_get_default_setting`.

Reply:
I changed on Java side to use untranslated settings names for preference keys.